.github/workflows/checks.yml

```diff
@@ -44,6 +44,6 @@ jobs:
     - name: isort
       run: isort . --check --diff
     - name: mypy
-      run: mypy .
+      run: awk '/\[mypy-torch\.\*\]/{exit} {print}' setup.cfg | mypy --config-file=/dev/stdin .
```

Comment:
I think this is a bit buggy because people may not notice that `setup.cfg` is cut off at the line after `[mypy-torch.*]`.

With this command, if we add any additional lines after `[mypy-torch.*]`, they will be ignored at the test time.

I would recommend adding inline markers to the start and the end of the lines that should be ignored.
Then it is not gonna be a complicated command and easier to circumvent the issue.

For example, if we change `setup.cfg` as follows:

```cfg
# IGNORE MYPY TORCH START (This line is used in checks.yml)
[mypy-torch.*]
# Any `torch`-related modules will be replaced with `typing.Any` at type checking.
follow_imports = skip
# IGNORE MYPY TORCH END (This line is used in checks.yml)
```

the line would look like this:

```shell
$ sed '/# IGNORE MYPY TORCH START/,/# IGNORE MYPY TORCH END/d' setup.cfg
```

By doing so, the shell command itself is clearer and more understandable by a glance.

Reply:
Agreed, and I confirmed your suggestion also works as intended. I changed the code.

Reply:
Thank you for the update!
Could you also change formats.sh?

Reply:
I think it is not needed to change formats.sh, because the behavior of `$ mypy .` and `$ bash formats.sh` should match in the user's local environment.

Reply:
It's probably right that this change might be confusing.
If necessary, we can come back to here when we modify `GPSampler`-related modules.